### PR TITLE
Fix CSRF config

### PR DESCRIPTION
**Pull request by koponkin**
## Fix https://app.clubhouse.io/vgs/story/9031/fix-csrf-in-django-sample-app

Add Nginx config
add Nginx as a service in docker-compose
Create a follow-up task to improve Readme
https://app.clubhouse.io/vgs/story/11359/change-vgs-django-sample-app-readme

### CODE REVIEW

File: idVerification/settings.py [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
 ALLOWED_HOSTS = ['*']
 
-INBOUND_ROUTE ='https://tntdwsvgf8u.SANDBOX.verygoodproxy.com'
-OUTBOUND_ROUTE ='https://USkvZxL8BJZf4RMdYjKmQzaN:df991a3d-57a4-4cbb-ba79-ed90f96e2b45@tntdwsvgf8u.SANDBOX.verygoodproxy.com:8080'
+# INBOUND_ROUTE ='https://tntvsu7b08w.SANDBOX.verygoodproxy.com'
+OUTBOUND_ROUTE ='https://USju2KiSSDqXBSvifhzSYkQz:074f9f4b-a5fc-4cb1-838b-8763107c6ac9@tntvsu7b08w.sandbox.verygoodproxy.com:8080'
```
Comment (osklyarenko) [inline idVerification/settings.py]: why did we change it?

File: app/views.py [hunk 2]
```diff
@@ -89,11 +88,10 @@ def json_serial(obj):
     raise TypeError("Type %s not serializable" % type(obj))
 
 
-@csrf_exempt
```
Comment (osklyarenko) [inline app/views.py]: why was it disabled?

File: idVerification/settings.py (hunk 1 above)
Comment (koponkin) [reply idVerification/settings.py]: There are my vault credentials. I use them to test app.

File: app/views.py (hunk 2 above)
Comment (koponkin) [reply app/views.py]: csrf uses 2 token for validation.
- token as a hidden form field
- token as a browser cookie
but when i send request (submit form) to our reverse proxy browser do not add cookies to this request. It was a workaround for POC.

File: nginx/django_nginx.conf.template [hunk 3]
```diff
@@ -0,0 +1,12 @@
+server {
+   listen       8001;
+   server_name  django_server_nginx;
+
+   location / {
+       proxy_pass ${DJANGO_APP_URL};
```
Comment (osklyarenko) [inline nginx/django_nginx.conf.template]: why not through VGS?

File: app/views.py [hunk 4]
```diff
@@ -89,11 +88,10 @@ def json_serial(obj):
...
 def add(request):
     ssn = request.POST['SSN']
     driver_license_number = request.POST['driver_license_number']
     pii_data = PiiData(social_security_number=ssn, driver_license_number=driver_license_number,
                        pub_date=datetime.now())
     pii_data.save()
-    return HttpResponse(str(pii_data.id))
+    return HttpResponseRedirect("/app/" + str(pii_data.id) + "/")
```
Comment (osklyarenko) [inline app/views.py]: was it a bug that you fixed

File: nginx/django_nginx.conf.template (hunk 3 above)
Comment (koponkin) [reply nginx/django_nginx.conf.template]: I used VGS proxy for requests with PII data

File: app/views.py (hunk 4 above)
Comment (koponkin) [reply app/views.py]: I used ajax on UI for make request for this endpoint and in `onSuccess` method I changed the current page. For now, I do not need ajax request but I have to change page. So I use redirect for it.

Comment (osklyarenko) [reply app/views.py]: got it.

File: idVerification/settings.py (hunk 1 above)
Comment (osklyarenko) [reply idVerification/settings.py]: correct. but i don't think this should be a part of the PR

Comment (koponkin) [reply idVerification/settings.py]: Reverted to previous routes

File: app/views.py (hunk 2 above)
Comment (osklyarenko) [reply app/views.py]: got it.

**Comment by osklyarenko** [comment]
[ch9031]

**Pull request merged**